addons/mail/static/tests/discuss/core/web/command_palette.test.js

```diff
@@ -113,20 +111,12 @@ test("Max 3 most recent channels in command palette of Discuss app with #", asyn
 
 test("only partners with dedicated users will be displayed in command palette", async () => {
     const pyEnv = await startServer();
-    const partnerId = pyEnv["res.partner"].create({ name: "test user" });
-    pyEnv["discuss.channel"].create({
-        name: "TestChanel",
-        channel_member_ids: [
-            Command.create({ partner_id: serverState.partnerId }),
-            Command.create({ partner_id: partnerId }),
-        ],
-        channel_type: "chat",
-    });
+    const demoUid = pyEnv["res.users"].create({ name: "Demo" });
+    pyEnv["res.partner"].create({ name: "Demo", user_ids: [demoUid] });
+    pyEnv["res.partner"].create({ name: "Portal" });
     await start();
     triggerHotkey("control+k");
     await insertText(".o_command_palette_search input", "@");
-    advanceTime(commandSetupRegistry.get("@").debounceDelay);
-    await contains(".o_command_name", { text: "Mitchell Admin" });
-    await contains(".o_command_name", { text: "OdooBot" });
```

Comment:
Both Admin and OdooBot need to be still shown and asserted in this test.

I'm suggesting to add an extra assert about number of results.